src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+This package has [not] been tested [much] with Python 2.7, 3.4 and 3.5.
+
+Commands to manage Azure container registries
```

Comment:
Agreed on the global arguments. @SajayAntony We still want to keep the readme, right?